Brisk/AppDelegate.swift

```diff
@@ -70,4 +73,28 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         self.statusItem?.remove()
         self.statusItem = nil
     }
+
+    private func registerRadarURLs() {
+        let eventManager = NSAppleEventManager.shared()
+        let selector = #selector(handleGetURLEvent(event:withReplyEvent:))
+        eventManager.setEventHandler(self, andSelector: selector,
+                                     forEventClass: AEEventClass(kInternetEventClass),
+                                     andEventID: AEEventID(kAEGetURL))
+    }
+
+    @objc
+    private func handleGetURLEvent(event: NSAppleEventDescriptor, withReplyEvent: NSAppleEventDescriptor) {
+        let radarID = event.paramDescriptor(forKeyword: keyDirectObject)?.stringValue.flatMap(radarID(from:))
```

Comment:
I would guard the link separately here so you can put it in the alert window message